rc/detection/modeline.kak

```diff
@@ -55,6 +55,7 @@ define-command -hidden modeline-parse-impl %{
             printf %s\\n "set-option buffer ${key} ${value}"
         }
 
+        type_selection=""
```

Comment:
I'd prefer to `exit 1` in the default case (line 61) and remove the test at line 63